### PR TITLE
LPS-107703 Adapt test

### PR DESCRIPTION
**Pull request by ruben-pulido**

### CODE REVIEW

File: modules/apps/headless/headless-delivery/headless-delivery-impl/rest-openapi.yaml [hunk 1]
```diff
@@ -949,6 +956,92 @@ components:
...
+                href:
+                    type: string
+            type: object
+        InlineValue:
+            properties:
+                value:
+                    type: string
+                value_i18n:
+                    additionalProperties:
+                        type: object
```
Comment (nhpatt) [inline modules/apps/headless/headless-delivery/headless-delivery-impl/rest-openapi.yaml]: It should be map of string, string, additionalProprties, type: string

File: modules/apps/headless/headless-delivery/headless-delivery-impl/rest-openapi.yaml [hunk 2]
```diff
@@ -1716,6 +1916,33 @@ components:
                         The title of the content.
                     readOnly: true
                     type: string
+        RowDefinition:
+            properties:
+                gutters:
+                    type: boolean
+                numberOfColumns:
+                    format: int32
```
Comment (nhpatt) [inline modules/apps/headless/headless-delivery/headless-delivery-impl/rest-openapi.yaml]: if it's an int, format is not needed

File: modules/apps/headless/headless-delivery/headless-delivery-impl/rest-openapi.yaml [hunk 3]
```diff
@@ -1716,6 +1916,33 @@ components:
...
+            properties:
+                backgroundColorCssClass:
+                    type: string
+                backgroundImage:
+                    $ref: "#/components/schemas/FragmentImage"
+                containerType:
+                    enum: [Fixed, Fluid]
+                    type: string
+                paddingBottom:
+                    format: int32
```
Comment (nhpatt) [inline modules/apps/headless/headless-delivery/headless-delivery-impl/rest-openapi.yaml]: =

File: modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/PageDefinitionDTOConverter.java [hunk 4]
```diff
@@ -0,0 +1,592 @@
...
+					layoutPageTemplateEntry, layout,
+					dtoConverterContext.getLocale());
+				settings = _toSettings(layout, dtoConverterContext.getLocale());
+				uuid = layoutPageTemplateEntry.getUuid();
+			}
+		};
+	}
+
+	private String _getFragmentCollectionName(
+		FragmentEntry fragmentEntry, String rendererKey, Locale locale) {
```
Comment (nhpatt) [inline modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/PageDefinitionDTOConverter.java]: he likes to sort parameters alphabetically too

File: modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/PageDefinitionDTOConverter.java [hunk 5]
```diff
@@ -0,0 +1,592 @@
...
+
+				return fragmentCollectionContributor.getName(locale);
+			}
+		}
+
+		return null;
+	}
+
+	private FragmentEntry _getFragmentEntry(
+		long fragmentEntryId, String rendererKey, Locale locale) {
```
Comment (nhpatt) [inline modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/PageDefinitionDTOConverter.java]: =

File: modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/PageDefinitionDTOConverter.java [hunk 6]
```diff
@@ -0,0 +1,592 @@
...
+			_fragmentEntryLinkLocalService.fetchFragmentEntryLink(
+				GetterUtil.getLong(
+					configJSONObject.getString("fragmentEntryLinkId")));
+
+		JSONObject editableValuesJSONObject = JSONFactoryUtil.createJSONObject(
+			fragmentEntryLink.getEditableValues());
+
+		String portletId = editableValuesJSONObject.getString("portletId");
+
+		String rendererKey = fragmentEntryLink.getRendererKey();
```
Comment (nhpatt) [inline modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/PageDefinitionDTOConverter.java]: sort by use

File: modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/PageDefinitionDTOConverter.java [hunk 7]
```diff
@@ -0,0 +1,592 @@
...
+
+		FragmentEntry fragmentEntry = _getFragmentEntry(
+			fragmentEntryLink.getFragmentEntryId(), rendererKey, locale);
+
+		return new FragmentDefinition() {
+			{
+				setFragmentCollectionName(
```
Comment (nhpatt) [inline modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/PageDefinitionDTOConverter.java]: why not fragmentCollectionName =

File: modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/PageDefinitionDTOConverter.java [hunk 8]
```diff
@@ -0,0 +1,592 @@
...
+								new long[] {0L})));
+				setFragmentName(
+					_getFragmentName(
+						fragmentEntry, portletId, rendererKey, locale));
+			}
+		};
+	}
+
+	private LookAndFeel _toLookAndFeel(
+		UnicodeProperties typeSettingsProperties) {
```
Comment (nhpatt) [inline modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/PageDefinitionDTOConverter.java]: convention :(

File: modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/PageDefinitionDTOConverter.java [hunk 9]
```diff
@@ -0,0 +1,592 @@
...
+			String key = entry.getKey();
+
+			if (key.startsWith("lfr-theme:")) {
+				themeSettingsProperties.setProperty(key, entry.getValue());
+			}
+		}
+
+		return new LookAndFeel() {
+			{
+				setThemeSettings(themeSettingsProperties);
```
Comment (nhpatt) [inline modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/PageDefinitionDTOConverter.java]: use =

File: modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/PageDefinitionDTOConverter.java [hunk 10]
```diff
@@ -0,0 +1,592 @@
...
+			}
+		};
+	}
+
+	private Map<String, Object> _toMap(
+		JSONObject configurationValuesJSONObject) {
```
Comment (nhpatt) [inline modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/PageDefinitionDTOConverter.java]: just jsonObject

File: modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/PageDefinitionDTOConverter.java [hunk 11]
```diff
@@ -0,0 +1,592 @@
...
+					String key = iterator.next();
+
+					put(key, configurationValuesJSONObject.get(key));
+				}
+			}
+		};
+	}
+
+	private MasterPage _toMasterPage(long masterLayoutPlid, Locale locale) {
+		Layout masterLayout = _layoutLocalService.fetchLayout(masterLayoutPlid);
```
Comment (nhpatt) [inline modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/PageDefinitionDTOConverter.java]: just layout

File: modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/PageDefinitionDTOConverter.java [hunk 12]
```diff
@@ -0,0 +1,592 @@
...
+					setName(masterLayout.getName(locale));
+				}
+			};
+		}
+
+		return null;
+	}
+
+	private PageElement _toPageElement(
+			JSONObject jsonObject, JSONObject itemsJSONObject, Locale locale)
```
Comment (nhpatt) [inline modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/PageDefinitionDTOConverter.java]: sort

File: modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/PageDefinitionDTOConverter.java [hunk 13]
```diff
@@ -0,0 +1,592 @@
...
+	}
+
+	private PageElement _toPageElement(
+			JSONObject configJSONObject, String type, Locale locale)
+		throws JSONException {
+
+		if (type.equals("column")) {
+			return new PageElement() {
+				{
+					setDefinition(_toColumnDefinition(configJSONObject));
```
Comment (nhpatt) [inline modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/PageDefinitionDTOConverter.java]: use =, setters only when it has several lines (ifs, variables)

File: modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/PageDefinitionResourceImpl.java [hunk 14]
```diff
@@ -27,4 +34,35 @@
 	scope = ServiceScope.PROTOTYPE, service = PageDefinitionResource.class
 )
 public class PageDefinitionResourceImpl extends BasePageDefinitionResourceImpl {
+
+	@Override
+	public PageDefinition getPageTemplate(Long pageTemplateId)
```
Comment (nhpatt) [inline modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/PageDefinitionResourceImpl.java]: method returns a pageDefinnition

Comment (ruben-pulido) [reply modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/PageDefinitionResourceImpl.java]: Method name is generated

**Comment by ruben-pulido** [comment]
https://github.com/nhpatt/liferay-portal/pull/203

**Pull request closed without merging**